### PR TITLE
Unifying GUI

### PR DESCRIPTION
**Pull request by dominikbach**
Stylises the Matlabbatch more uniformly.

Changes proposed in this pull request:
- common selector functions for items that repeat across modules: 
```
pspm_cfg_selector_channel
pspm_cfg_selector_channel_action
pspm_cfg_selector_data_design
pspm_cfg_selector_datafile
pspm_cfg_selector_filter
pspm_cfg_selector_norm
pspm_cfg_selector_outputfile
pspm_cfg_selector_overwrite
pspm_cfg_selector_python
pspm_cfg_selector_timeunits
```
- common dependency creators for 3 types of dependencies: data files, model files, channel index
```
pspm_cfg_vout_modelfile
pspm_cfg_vout_outchannel
pspm_cfg_vout_outfile
```
- separate run functions for all cfg functions
- stylistic changes
- removed some deprecated items
- some bugfixes and improvements in the main PsPM functions

### CODE REVIEW

**Comment by teddychao** [comment]
Hi @dominikbach apologies for uploading two commits to this PR.
1. I need to add a comment to `src/pspm_cfg/pspm_cfg_data_design_selector.m`. This file was not changed in this PR so I have to add something to reveal it in change list for commenting. I only added a space that can be deleted later if you wish.
2. I fixed the conflicts that are caused by PR 659. These changes only attempted to fix the conflicts and have no modifications to any change that is introduced in this PR.

**Review by teddychao — APPROVED** [review]

**Comment by teddychao** [comment]
I approved this PR as the test in GUI at my side seems to operate well. I will continue testing in the PR 721 later.

File: src/pspm_cfg/pspm_cfg_import.m
[the remark this answers is not in the document]
Comment (dominikbach) [reply src/pspm_cfg/pspm_cfg_import.m:1]: fixed.

File: src/pspm_cfg/pspm_cfg_data_design_selector.m [hunk 1]
```diff
@@ -17,7 +17,7 @@
         % datafile
         model.datafile{iSession,1} = job.session(iSession).datafile{1};
         % missing epochs
-        if isfield(job.session(iSession).missing,'epochfile')
+        if isfield(job.session(iSession).missing,'epochfile') 
             model.missing{1,iSession} = job.session(iSession).missing.epochs.epochfile{1};
```
[the remark this answers is not in the document]
Comment (dominikbach) [reply src/pspm_cfg/pspm_cfg_data_design_selector.m:21]: fixed,

File: src/pspm_cfg/pspm_cfg_selector_python.m [hunk 2]
```diff
@@ -1,4 +1,4 @@
-function out = pspm_cfg_python(varargin)
+function out = pspm_cfg_selector_python(varargin)
 % ● Description
 %   pspm_cfg_python is a function that provides UI controls for
```
[the remark this answers is not in the document]
Comment (dominikbach) [reply src/pspm_cfg/pspm_cfg_selector_python.m:3]: fixed.

**Pull request merged**